### DISCUSSION TITLE
Github profile

### DISCUSSION THREAD

**Discussion by elakoc** [community/community#51607] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I hide my Github profile?

**Comment by ramazancetinkaya** [comment, marked as answer]
You can follow these steps:

1) https://github.com/settings/profile

2) Contributions & Activity

Make sure "Make profile private and hide activity" is checked.

Enabling this will hide your contributions and activity from your GitHub profile and from social features like followers, stars, feeds, leaderboards and releases.

**Reply by Xwilarg** [reply]
Is there a way to hide from your profile what repository you own without setting them as private? (So I don't loose all the stars and stuff)
